packages/email-editor/src/web/EmailEditor.module.css

```
@@ -0,0 +1,275 @@
+.surface {
+  --email-editor-border: hsl(var(--border, 214 32% 91%));
+  --email-editor-muted: hsl(var(--muted, 210 40% 96%));
+  --email-editor-muted-foreground: hsl(var(--muted-foreground, 215 16% 47%));
+  --email-editor-foreground: hsl(var(--foreground, 222 47% 11%));
+  --email-editor-background: hsl(var(--background, 0 0% 100%));
+  --email-editor-ring: hsl(var(--ring, 215 20% 65%));
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)